<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="Result">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConversion">
    <property name="title">
     <string>Conversion</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCpp"/>
    <addaction name="actionExecuter"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionConfigurer"/>
   </widget>
   <addaction name="menuConversion"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionF">
   <property name="text">
    <string>f</string>
   </property>
  </action>
  <action name="actionCpp">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionExecuter">
   <property name="text">
    <string>Executer</string>
   </property>
  </action>
  <action name="actionConfigurer">
   <property name="text">
    <string>Configurer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
